### ISSUE TITLE
ShareX Microsoft Store App - Issue with ffmpeg download

### ISSUE THREAD

**Issue by s-malik** [ShareX/ShareX#3877]
I just installed ShareX Microsoft Store App. I can not download ffmpeg.exe from the program. The Download button is greyed out.

![ffmpeg download](https://user-images.githubusercontent.com/46628021/51074909-72beab00-164a-11e9-95cd-b0c04a3c7ce5.png)

**Comment by Jaex** [comment]
Did you tried checking "Use custom path"?

**Comment by s-malik** [comment]
I tried, but that location (%MyDocument%\ShareX\Tools) does not even exist. Then, I uninstalled the ShareX Microsoft Store app, removed all related files and folders and installed ShareX desktop version. No issue downloading ffmpeg and that location  (%MyDocument%\ShareX\Tools) does exist in the desktop version.

I like ShareX, especially the virtual audio and video capture option. I have tried some other very well known screenshot/screen recorder programs and none of them are even close to ShareX. Thank you for your efforts to provide such a nice program.
